docs/azure/intro.md

```diff
@@ -61,6 +63,14 @@ Azure offers several services for storing your data depending on your needs. The
 
 If you use Visual Studio, you can add support for certain Azure services to your projects. The **Connected Services** dialog in Visual Studio provides an easy way to add all the required references, connection code, and configuration settings to your projects. Some commonly used Azure services are supported out of the box, such as [Storage](/azure/vs-azure-tools-connected-services-storage), [Azure Active Directory](/azure/active-directory/develop/vs-active-directory-add-connected-service) authentication, [Azure Key Vault](/azure/key-vault/vs-key-vault-add-connected-service), and [Cognitive Services](/azure/cognitive-services/) such as [Computer Vision](/azure/cognitive-services/computer-vision/vs-computer-vision-connected-service). More services, including third-party services, are available as extensions in the [Visual Studio Marketplace](https://marketplace.visualstudio.com/search?term=connected%20service&target=VS&category=Tools&vsVersion=&subCategory=All&sortBy=Relevance).
 
+## Using the Azure SDK for .NET
+
+If you're using the Azure SDK for .NET to consume or manage your Azure resources, please note the following:
+
+* **Authentication**: Many libraries in the SDK use a common authentication infrastructure, while some libraries use authentication mechanisms specific to the service they're consuming. See [Authentication with the Azure SDK for .NET](authentication.md) for further details.
```

Comment:
I don't think libraries are "consuming" services. I would say libraries provide access to services.

Reply:
Agreed